<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventsWindow</class>
 <widget class="QMainWindow" name="EventsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="FastPositionsLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Fast Positions in Phase</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>356</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="positionsButtonsLayout">
     <item>
      <widget class="QPushButton" name="getPositionsButton">
       <property name="text">
        <string>Get Positions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finalizePositionsButton">
       <property name="text">
        <string>Finalize Positions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetPositionsButton">
       <property name="text">
        <string>Reset Positions</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="SlowPositionsLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Slow Positions in Phase</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="presetsBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>621</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Presets and Channels</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="presetsLayout">
      <item>
       <widget class="QPushButton" name="addPresetButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removePresetButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="channelExposureList">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>231</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="channelExposureLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="channelLabel">
        <property name="text">
         <string>Channel:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="presets"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="exposureLabel">
        <property name="text">
         <string>Exposure(ms):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="exposure"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="timeIntervalLabel">
        <property name="text">
         <string>Time Interval(s)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="minTimeIntervalSpinBox">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="nTimePointsLabel">
        <property name="text">
         <string>No of TimePoints</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="nTimePointsSpinBox">
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>200</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Always add phase preset First</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>616</width>
      <height>194</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="positionListsLayout">
     <item>
      <widget class="QListWidget" name="fastPositions"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="sendToSlowButton">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendToFastButton">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="slowPositions"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>640</y>
      <width>318</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="eventButtonsLayout">
     <item>
      <widget class="QPushButton" name="constructEventsButton">
       <property name="text">
        <string>Construct Events</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetEventsButton">
       <property name="text">
        <string>Reset Events</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeWindowButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
